components/script/dom/webglrenderingcontext.rs

```diff
@@ -1039,6 +1039,147 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10
+    fn Uniform2fv(&self,
+                  _cx: *mut JSContext,
+                  uniform: Option<&WebGLUniformLocation>,
+                  data: Option<*mut JSObject>) {
+        let data = match data {
```

Comment:
That refers to the `WebGLUniformLocation` variable, which is ignored if it's null in `Uniform2f()`... That check should be repeated here though, since in the case `null, null` were registering an error when we should not.